docs/pages/guides/dom-components.mdx

```diff
@@ -303,6 +305,8 @@ The `router.canGoBack()` and `router.canDismiss()` functions are also unsupporte
 
 Avoid using standard web `<a />` anchor elements for navigation as these will change the DOM component origin in a way that users may not be able to navigate back from. Prefer launching `WebBrowser`s if you want to present external websites.
 
+Because DOM components cannot render native children, layout routes (`_layout`) can never be DOM components. You can render DOM components from layout routes to create headers, backgrounds, etc. but the layout route itself should always be native.
```

Comment:
```suggestion
Since DOM components cannot render native children, layout routes (`_layout`) can never be DOM components. You can render DOM components from layout routes to create headers, backgrounds, and so on, but the layout route itself should always be native.
```